ocrd_tool.md

```diff
@@ -19,6 +19,27 @@ Optionally, workflow processors can be notified that this file is potentially
 large and static (e.g. a fixed dataset or a precomputed model) and should be cached
 indefinitely after download by setting the `cacheable` property to `true`.
 
+The filename itself, i.e. the concrete value `<fpath>` of a file parameter
+should be resolved in the following way:
+
+* If `<fpath>` is an `http`/`https` URL: Download to a temporary directory (if
+  `cacheable==False`) or a semi-temporary cache directory (if `cacheable==True`)
+* If `<fpath>` is an absolute path: Use as-is.
+* If `<fpath>` is a relative path, try resolving the following paths and return
+  the first one found if any, raise a `FileNotFoundError` otherwise:
+  * `$CWD/<fpath>`
+  * If an environment variable is defined that has the name of the processor in
+    upper-case and with `-` replaced with `-` and followed by `_PATH` (e.g. for a processor
+    `ocrd-dummy`, the variable would need to be called `OCRD_DUMMY_PATH`):
+    * Split the variable value at `:` and try to resolve by prepending each token
+      with `<fname>`
+  * `$VIRTUAL_ENV/share/<name-of-processor>/<fpath>`
```

Comment:
This one is quite Python-specific, and even venv-specific (which our Docker images don't use). Plus this would already be (functionally) equivalent to the last list item, wouldn't it?

Reply:
It is Python-specific but local ocrd_all will install also the non-python code to `$VIRTUAL_ENV`. The really python-specific variant would be `os.path.join(sysconfig.get_path('data'), 'share', '<name-of-processor>', '<fpath>))`. And python-specific is okay IMHO because it offers python developers an additional way to bundle data with a processor while still allowing users to store their processor-specific files in clearly defined locations.

Reply:
I fully agree.